TRACEABILITY.md

```diff
@@ -0,0 +1,43 @@
+# Traceability utility
+
+This utility provides several methods to facilitate traceability of specs against test cases. 
```

Comment:
Again, "traceability between specs and test cases" or "traceability of specs with test cases"